src/main/java/gov/usds/case_issues/config/SecurityConfig.java

```diff
@@ -54,6 +55,7 @@ protected void configure(HttpSecurity http) throws Exception {
 			.csrf()
 				.ignoringRequestMatchers(AnyRequestMatcher.INSTANCE)
 		;
+		http.exceptionHandling().accessDeniedHandler(new CustomAccessDeniedHandler());
```

Comment:
Oooh, I didn’t realize we were in here. I think we can eliminate the redundant logging, and make it more obvious why we’re doing this. 

If I’m not crazy, and/or just sleep-deprived, you can restrict the application of some security configuration based on the URL using something like `http.antMatcher`, so we could do `http.antMatcher(resourceBase).exceptionHandling...` and only get this logging when we want it. 

Also, in the general case you can use the fluent API on `http` and save semicolons, but that may not combine with the thing I mentioned above, and that’s obviously more important.

Reply:
I don't think it is possible to limit the exception handling with the built in  path matching. Only way I can see is to check the path manually in our custom handler and call the default handler if the path does not start with `resource` but I dont think that is worth it right now